src/workerfarm/WorkerFarm.js

```diff
@@ -23,10 +22,14 @@ class WorkerFarm extends EventEmitter {
     this.warmWorkers = 0;
     this.children = new Map();
     this.callQueue = [];
-
-    this.localWorker = require(this.options.workerPath);
-    this.run = this.mkhandle('run');
-
+    this.workerFunctions = {};
+    
+    this.localWorker = require('./childBootstrap');
+    this.registerWorkerFunctions(this.options.workerPath || '../worker', [
+      'run',
+      'init'
```

Comment:
Maybe we could autodetect these instead of needing to list them? Basically `Object.keys(require(workerPath))`?

Reply:
I can change it, but think it'll be more precise if it has to be defined when being called.
For example if a worker has more than just the worker functions as exports we don't add a handler for all of them.
If we document this eventually it should be fine to use export keys I guess.